docs/guides/admin/docs/releasenotes.md

```diff
@@ -42,6 +42,9 @@ Configuration changes
 API changes
 -----------
 
+Due to [MH-13392](https://opencast.jira.com/browse/MH-13392):
```

Comment:
This is confusing as this PR does not change this request for existing External API versions so there is no API change here. I suggest to just omit this.

Reply:
Ok, will remove it.